contracts/src/verifier/Verifier.sol

```diff
@@ -36,7 +36,7 @@ contract Verifier is Ownable2Step {
      * @param proof Zk proof of the passkey signature
      */
 	function verifyPasskeySignature(address account, bytes memory proof, bytes memory challenge) external returns(bool) {
-        (bool success,) = snarkVerifier.call(abi.encode(publicKeys[account], proof));
+        (bool success,) = snarkVerifier.call(abi.encode(publicKeys[account], proof, challenge));
```

Comment:
it should be this (i think the first line will have a syntax error but you get the idea)
```suggestion
        uint256[] memory pubInputs = [publicKeys[account], challenge];
        (bool success,) = snarkVerifier.call(abi.encode(pubInputs, proof)));
```